helpers/parse_helpers.go

```diff
@@ -88,7 +88,7 @@ func processRecipientKeys(recipients []string) ([][]byte, [][]byte, [][]byte, []
 func processx509Certs(keys []string) ([][]byte, error) {
 	var x509s [][]byte
 	for _, key := range keys {
-		tmp, err := ioutil.ReadFile(key)
+		tmp, err := ioutil.ReadFile(strings.Split(key, ":")[0])
```

Comment:
Hmm... what use case would the password for certs be for?

Reply:
There's no use case. The issue is `keys` can come in the format of filename:pass=<password> and so we have to split them here.

Reply:
I think the other process functions just ignore it if it doesn't fit what it's expecting, since its passed into multiple process functions and picked out by the one that matches. I think this is ok though since it only affects the case where a semicolon is part of the filename of the cert, but it is bad practice so it should not happen anyway...

i'm curious, did you encounter an issue here when you were using it?

Reply:
> i'm curious, did you encounter an issue here when you were using it?

I had tried to use it with imgcrypt (after disabling all gpg related testing) and this particular line caused a failure since it passes a password in and the file cannot be read unless it is split: https://github.com/containerd/imgcrypt/blob/master/script/tests/test_encryption.sh#L546

Reply:
Also, each member of the keys list is always split here: https://github.com/containers/ocicrypt/blob/master/helpers/parse_helpers.go#L156 . It's the same list...